testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxBoundedContext.java

```diff
@@ -246,7 +228,7 @@ public static MultitenantBlackBoxContext multiTenant(EventEnricher enricher) {
     /**
      * Creates a multitenant instance with the specified name and enricher.
      */
-    public static MultitenantBlackBoxContext multiTenant(String name, EventEnricher enricher) {
+    private static MultitenantBlackBoxContext multiTenant(String name, EventEnricher enricher) {
```

Comment:
I assume that the rest of similar factory methods will become `private` or even get removed in part 2.

Reply:
Yes, I intend to remove them at all in part 2. I'll make the deprecated in this part.

Reply:
As discussed vocally, I'll revert it and address them all in p2.